### ISSUE TITLE
Authorization header missing

### ISSUE THREAD

**Issue by andreikashyn** [lepicekmichal/SignalRKore#5]
Hello again, unfortunately found another bug... 
When `skipNegotiate` is set to `true` then neither the accessToken nor the custom headers are set when connecting to the URL.
Example 1: 
```
signalRConnection = HubConnectionBuilder.create("https://example.com/notificationHub") {
        headers = mutableMapOf(HttpHeaders.Authorization to "Bearer $myAccessToken")
        skipNegotiate = true
        transportEnum = TransportEnum.WebSockets
        logger = Logger { severity, message, cause ->
            AppContext.logger.d("SignalR", "Log from SignalR: $severity - message: $message, cause: $cause")
        }
    }
```
Example 2:
```
signalRConnection = HubConnectionBuilder.create("https://example.com/notificationHub") {
        accessToken = myAccessToken
        skipNegotiate = true
        transportEnum = TransportEnum.WebSockets
        logger = Logger { severity, message, cause ->
            AppContext.logger.d("SignalR", "Log from SignalR: $severity - message: $message, cause: $cause")
        }
    }
```
In both cases Authorization header is missing in the final request

**Comment by andreikashyn** [comment]
I believe the bug is with this extension function or the way it is used, because it returns `Headers` object that is not used/set inside of `WebSocketTransport`.

I think it should be something like this:
```
headers {
                this@WebSocketTransport.headers.forEach { (key, value) ->
                    append(key, value)
                }
            }
```

**Comment by lepicekmichal** [comment]
thank you for reporting, fixed in [0.8.4](https://github.com/lepicekmichal/SignalRKore/releases/tag/0.8.4)
